server/resp/src/main/java/org/infinispan/server/resp/meta/MetadataRepository.java

```diff
@@ -0,0 +1,25 @@
+package org.infinispan.server.resp.meta;
+
+/**
+ * Stores RESP related metadata and statistics.
+ *
+ * @author José Bolina
+ * @since 15.0
+ */
+public final class MetadataRepository {
```

Comment:
I assume we will eventually put more stuff in this class?

Reply:
Hopefully, yes. If not for something related to RESP, it could be metrics we add.